Gemfile

```diff
@@ -158,7 +159,7 @@ gem 'html-pipeline'
 gem 'ruby-openid'
 gem 'slim-rails', '~> 3.2'
 
-gem 'draper', '~> 3.1'
+gem 'draper', '~> 4.0.2'
```

Comment:
Fixing:
```
/home/dmayorov/.asdf/installs/ruby/3.1.4/lib/ruby/gems/3.1.0/gems/activesupport-6.1.7.8/lib/active_support/core_ext/module/delegation.rb:173:in `delegate': Delegation needs a target. Supply a keyword argument 'to' (e.g. delegate :hello, to: :greeter). (ArgumentError)
```

`4.0.2` is the first version with Ruby 3 compatibility: https://github.com/drapergem/draper/blob/master/CHANGELOG.md#402---2021-05-27